core/src/test/java/org/elasticsearch/search/functionscore/FunctionScoreBackwardCompatibilityIT.java

```diff
@@ -62,7 +61,7 @@ public void testSimpleFunctionScoreParsingWorks() throws IOException, ExecutionE
                         .endObject()
                         .endObject()
                         .endObject()
-                        .endObject()));
+                    .endObject()).setSettings("number_of_shards", 1));
```

Comment:
Sorry, that was burried in a commit message:
`FunctionScoreBackwardCompatibilityIT` relied on tf/idf information but didn't
set the number of shards to 1 or use dfs_query_then_fetch. Now it uses one
shard.

I'll add it to a comment here if you approve.